src/lib/utils.ts

```diff
@@ -9,8 +9,11 @@ import { validators } from './validators';
  */
 export const validate = (typeName: string, data: any): boolean => {
   const validateFn: ValidateFunction | undefined = validators.get(typeName);
-  // TODO: ignore validation if no validator or throw ?
-  if (!validateFn) return data;
+
+  if (!validateFn) {
+    throw Error(`Validate function not defined for type [${typeName}]`);
```

Comment:
I would use `Validate function not defined for type '${typeName}'`, I don't like those [] but it's a personnal preference.

Reply:
doen